DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: ixpantia.introR
 Type: Package
-Title: Aprenda R con ejercicios interactivos del curso introducción a R de ixpantia.
+Title: Aprenda R con ejercicios interactivos del curso introducción a R de ixpantia
```

Comment:
Actually, this must be in title case, i.e. "Aprenda R con Ejercicios Interactivos ..."